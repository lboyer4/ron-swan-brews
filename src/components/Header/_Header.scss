.header {
	background-image: url('https://images.unsplash.com/photo-1504502350688-00f5d59bbdeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2550&q=80');
	background-position: center top;
  background-repeat: no-repeat;
  background-size: cover;
  height: 560px;
}