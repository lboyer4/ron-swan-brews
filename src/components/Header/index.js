import React from 'react';
import './_Header.scss'

const Header = () => {
	return (
		<header className="header">
			
		</header>
		)

}

export default Header;