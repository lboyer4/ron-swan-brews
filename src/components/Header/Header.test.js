import React from 'react';
import { shallow, mount } from 'enzyme';
import Header from '../Header';


describe('Header', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(
			<Header />
		);
	});

	it('should match snapshot', () => {
		expect(wrapper).toMatchSnapshot()
	});
});