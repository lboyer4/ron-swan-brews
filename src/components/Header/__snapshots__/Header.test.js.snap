// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header
  className="header"
>
  <Connect(Icon) />
  <Connect(Form) />
</header>
`;
